DAOS-12412 cart: Debug DFS test.

jgmoore-or opened the pull request:
Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

jgmoore-or committed:
DAOS-12412 cart: Debug DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12412 cart: Fix DFS tests.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/2/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/3/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/4/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/5/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/6/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/7/execution/node/512/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/8/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/10/execution/node/512/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/11/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+cc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/12/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/13/execution/node/512/log

jgmoore-or committed:
DAOS0-12412 cart: Fix DFS test.
Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/14/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/16/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/17/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/18/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/19/execution/node/512/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/20/execution/node/509/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/21/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/22/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix dfs test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/23/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/24/execution/node/214/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.4 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/24/execution/node/217/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/24/execution/node/220/log

jgmoore-or committed:
DAOS-12412 cart: Fix dfs test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:2

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/27/execution/node/509/log

jgmoore-or committed:
DAOS 12412 cart: Fix DFS test.

Merge branch 'master' into jgm/daos-12412

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:2

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/29/execution/node/513/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:3

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/30/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DfS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:4

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/31/execution/node/512/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:5

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/32/execution/node/512/log

jgmoore-or committed:
Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-94:1

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/33/execution/node/509/log

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/34/execution/node/195/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.4 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/34/execution/node/220/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/34/execution/node/198/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Test-provider: ucx+dc_x
PR-repos: mercury@PR-94:1

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented on src/object/srv_obj.c:
```diff
@@ -3740,7 +3740,11 @@ ds_obj_query_key_handler(crt_rpc_t *rpc, bool return_epoch)
 	D_ASSERT(okqo != NULL);
-
+/*
+	D_EMIT("Object query key invoked for OID: %lu,%lu\n",
```
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 commented on src/object/srv_obj.c:
```diff
@@ -3802,6 +3806,10 @@
 	if (rc != 0)
 		D_ERROR("send reply failed: "DF_RC"\n", DP_RC(rc));
+/*
+	D_EMIT("Object query key completed, with return code: %d, for OID: %lu,%lu\n",
```
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 replied:
(style)  Block comments use * on subsequent lines

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11673/35/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/35/execution/node/173/log

daosbuild1 commented:
Test stage Build RPM on EL 8 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/35/execution/node/338/log

daosbuild1 commented:
Test stage Build RPM on Leap 15.4 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/35/execution/node/341/log

daosbuild1 commented:
Test stage Build DEB on Ubuntu 20.04 completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/35/execution/node/288/log

jgmoore-or committed:
OS-12412 cart: Fix DFS test.

Quick-Functional: true
Test-tag: daos_core_test_dfs
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:5

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/38/execution/node/509/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/39/execution/node/509/log

jgmoore-or committed:
DAOS-12412 cart Fix DFS test.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium: true
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:5

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11673/40/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/40/execution/node/671/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:5

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 commented on src/object/srv_obj.c:
```diff
@@ -3802,6 +3806,10 @@ ds_obj_query_key_handler(crt_rpc_t *rpc, bool return_epoch)
 	if (rc != 0)
 		D_ERROR("send reply failed: "DF_RC"\n", DP_RC(rc));
+/*
+	D_EMIT("Object query key completed, with return code: %d, for OID: %lu,%lu\n",
```
(style)  Block comments use * on subsequent lines

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11673/41/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/41/execution/node/145/log

daosbuild1 commented:
Test stage Functional Hardware Large completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/41/execution/node/672/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/41/execution/node/761/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:6

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (dismissed):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11673/42/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/42/execution/node/145/log

jgmoore-or committed:
DAOS-12412 cart: Fix DFS test.

Skip-build-leap15-icc: true
Skip-build-el8-gcc: true
Skip-build-ubuntu20-rpm: true
Skip-func-test-vm: true
Skip-unit-tests: true
Skip-func-hw-test-medium-verbs-provider: true
Test-provider: ucx+dc_x
PR-repos: mercury@PR-98:7

Signed-off-by: Joseph Moore <joseph.moore@intel.com>

daosbuild1 reviewed (changes requested):
Style warning(s) for job https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-11673/43/
Please review https://wiki.hpdd.intel.com/display/DC/Coding+Rules

daosbuild1 commented:
Test stage checkpatch completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/43/execution/node/146/log

daosbuild1 commented:
Test stage Functional Hardware Medium completed with status FAILURE.  https://build.hpdd.intel.com//job/daos-stack/job/daos/view/change-requests/job/PR-11673/43/execution/node/762/log
